persistence.js

```diff
@@ -530,17 +466,17 @@ RedisPersistence.prototype.getWill = function (client, cb) {
 RedisPersistence.prototype.delWill = function (client, cb) {
   var key = 'will:' + client.brokerId + ':' + client.id
   var result = null
-  var pipeline = this._getPipeline()
-  pipeline.lrem(willKey, 0, key)
-  pipeline.getBuffer(key, function getClientWill (err, packet) {
+  var that = this
```

Comment:
Yes, del should have been inside the getBuffer after multi being removed, fixed it just now @GavinDmello